api/envoy/config/common/accesslog/v3/common.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package envoy.config.common.accesslog.v3;
```

Comment:
Ideally we refactor the existing ResponseFlagFilter to a separate file in v3 via `import public`, then move its package to v4alpha via a next package annotation. You should be able to find some examples of how this has been done for other such migrations.

Reply:
@htuch : PTAL, I have added v4alpha package for this...

Reply:
I was suggesting we continue to use `api/envoy/config/accesslog/v3/accesslog.proto` in v3, but move it to the common location in v4 by annotating with `(udpa.annotations.file_migrate).move_to_package`. In the PR, you still have a new proto for this in v3.